### PR TITLE
[com_contact] update component install mysql sql

### PR DESCRIPTION
**Pull request by andrepereiradasilva**
### Summary of Changes

Update com_contact create table install sql that is outdated.
### Testing Instructions

Code review.
Copied from here: https://github.com/joomla/joomla-cms/blob/staging/installation/sql/mysql/joomla.sql#L251
### Documentation Changes Required

None.

### CODE REVIEW

File: administrator/components/com_contact/sql/install.mysql.utf8.sql [hunk 1]
```diff
@@ -1,47 +1,52 @@
...
+  `created` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
+  `created_by` int(10) unsigned NOT NULL DEFAULT 0,
+  `created_by_alias` varchar(255) NOT NULL DEFAULT '',
+  `modified` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
+  `modified_by` int(10) unsigned NOT NULL DEFAULT 0,
   `metakey` text NOT NULL,
   `metadesc` text NOT NULL,
   `metadata` text NOT NULL,
-  `featured` tinyint(3) unsigned NOT NULL default '0' COMMENT 'Set if contact is featured.',
+  `featured` tinyint(3) unsigned NOT NULL DEFAULT 0 COMMENT 'Set if article is featured.',
```
Comment (jeckodevelopment) [inline administrator/components/com_contact/sql/install.mysql.utf8.sql]: article?

Comment (andrepereiradasilva) [inline administrator/components/com_contact/sql/install.mysql.utf8.sql]: fixed

**Comment by jeckodevelopment** [comment]
Also install.sql is affected.
I updated it in my 11614

**Comment by andrepereiradasilva** [comment]
thanks

**Pull request merged**